### ISSUE TITLE
Is it possible to add an auto suggest ?

### ISSUE THREAD

**Issue by 1-800-jono** [JFusco/react-tagging-input#1]
Great tag input component man!
I was wondering if is it possible to add an auto suggest based on a list?

**Comment by JFusco** [comment]
@jonathanphz 
Thanks a lot!

Not with the current implementation, but it's a good idea to implement - I accept pull requests as well if you want to give it a shot!

**Comment by 1-800-jono** [comment]
Yeah I was thinking about it - was wondering if I could just implement https://github.com/moroshko/react-autosuggest to the component. Need to look into it a bit.

**Comment by JFusco** [comment]
@jonathanphz 
Sounds like a good weekend project, I'll give it a shot this weekend
